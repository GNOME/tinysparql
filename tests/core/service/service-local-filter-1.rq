SELECT ?u {
  SERVICE <%s> {
    ?u nrl:indexed true;
       rdfs:label ?l
  }
  FILTER (STRSTARTS(?l, "Alb"))
}
