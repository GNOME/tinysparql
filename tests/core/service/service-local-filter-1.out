"http://tracker.api.gnome.org/ontology/v3/nmm#musicAlbum"
